specs/protocol/README.md

```diff
@@ -319,19 +276,29 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
 
-| field          | data type                                         | required | description                                                                           |
-|----------------|---------------------------------------------------|----------|---------------------------------------------------------------------------------------|
-| `offeringId`   | string                                            | Y        | Offering which Alice would like to get a quote for                                    |
-| `payinAmount`  | [`DecimalString`](#decimalstring)                 | Y        | Amount of payin currency you want in exchange for payout currency                     |
-| `claims`       | string[]                                          | Y        | an array of claims that fulfill the requirements declared in an [Offering](#offering) |
-| `payinMethod`  | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to send payin currency.                                  |
-| `payoutMethod` | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to receive payout currency.                              |
+| field          | data type                                         | required | description                                                                                                                 |
+|----------------|---------------------------------------------------|----------|-----------------------------------------------------------------------------------------------------------------------------|
+| `offeringId`   | string                                            | Y        | Offering for which Alice would like to get a quote                                                                          |
+| `payinAmount`  | [`DecimalString`](#decimalstring)                 | Y        | Amount of payin currency you want in exchange for payout currency                                                           |
+| `claims`       | string[]                                          | N        | An array of claims that fulfill the requirements declared in an [Offering](#offering). Excluded from the signature payload. |
+| `claimsHash`   | string                                            | N        | A hash [digest](#digest) of `claims`                                                                                        |
+| `payinMethod`  | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to send payin currency.                                                                        |
+| `payoutMethod` | [`SelectedPaymentMethod`](#selectedpaymentmethod) | Y        | Specify which payment method to receive payout currency.                                                                    |
+
+#### Unsigned Fields
```

Comment:
This name is possibly misleading. I would instead refer to it as `Disclosable Fields` as you are proposing to sign over hashes and then disclose the unsigned preimage.